stdlib/int64.gr

```diff
@@ -329,39 +610,56 @@ export let rec lxor = (x: Int64, y: Int64) => {
   ret
 }
 
-
+/**
+ * Counts the number of leading zeros in value.
+ *
+ * @param value: The value to inspect
+ * @returns Values count of leading zeros
+ * 
+ * @since v0.4.0
+ */
 @disableGC
-export let rec shl = (x: Int64, y: Int64) => {
-  let xv = WasmI64.load(WasmI32.fromGrain(x), 8n)
-  let yv = WasmI64.load(WasmI32.fromGrain(y), 8n)
-  let ptr = newInt64(WasmI64.shl(xv, yv))
+export let rec clz = (value: Int64) => {
+  let nv = WasmI64.load(WasmI32.fromGrain(value), 8n)
+  let ptr = newInt64(WasmI64.clz(nv))
   let ret = WasmI32.toGrain(ptr): Int64
-  Memory.decRef(WasmI32.fromGrain(x))
-  Memory.decRef(WasmI32.fromGrain(y))
-  Memory.decRef(WasmI32.fromGrain(shl))
+  Memory.decRef(WasmI32.fromGrain(value))
+  Memory.decRef(WasmI32.fromGrain(clz))
   ret
 }
 
+/**
+ * Counts the number of trailing zeros in value.
+ *
+ * @param value: The value to inspect
+ * @returns Values count of trailing zeros
+ * 
+ * @since v0.4.0
+ */
 @disableGC
-export let rec shr = (x: Int64, y: Int64) => {
-  let xv = WasmI64.load(WasmI32.fromGrain(x), 8n)
-  let yv = WasmI64.load(WasmI32.fromGrain(y), 8n)
-  let ptr = newInt64(WasmI64.shrS(xv, yv))
+export let rec ctz = (value: Int64) => {
+  let nv = WasmI64.load(WasmI32.fromGrain(value), 8n)
+  let ptr = newInt64(WasmI64.ctz(nv))
   let ret = WasmI32.toGrain(ptr): Int64
-  Memory.decRef(WasmI32.fromGrain(x))
-  Memory.decRef(WasmI32.fromGrain(y))
-  Memory.decRef(WasmI32.fromGrain(shr))
+  Memory.decRef(WasmI32.fromGrain(value))
+  Memory.decRef(WasmI32.fromGrain(ctz))
   ret
 }
 
+/**
+ * The number of 1-bits in value.
+ *
+ * @param value: The value to inspect
+ * @returns The number of 1-bits in its operand
```

Comment:
Don't use the description as the returns.

Reply:
```suggestion
 * @returns The amount of 1-bits in its operand
```
Since the above description will be changed, we can keep this. Though I want it to say "amount" instead of "number"